test/dynamo/test_aot_autograd_cache.py

```diff
@@ -58,6 +64,105 @@ def _clear_dynamo_and_codecache(self):
         torch._dynamo.reset()
         torch._inductor.codecache.PyCodeCache.cache_clear(purge=True)
 
+    @requires_triton()
+    @functorch_config.patch({"enable_autograd_cache": True})
+    @inductor_config.patch(
+        {
+            "fx_graph_cache": True,
```

Comment:
what's the story around the boolean configs for `fx_graph_cache` vs `enable_autograd_cache`? Do we expect fx_graph_cache to be `True` if the AOTAutograd cache is on?

Reply:
Yeah, if it's not on, it AOTAutogradCache will bypass